src/tests/simulation/src/slt.rs

```diff
@@ -59,16 +70,31 @@ impl SqlCmd {
 }
 
 fn extract_sql_command(sql: &str) -> SqlCmd {
-    let cmd = sql
-        .trim_start()
-        .split_once(' ')
-        .unwrap_or_default()
-        .0
-        .to_lowercase();
-    match cmd.as_str() {
-        "create" => SqlCmd::Create {
-            is_create_table_as: is_create_table_as(sql),
+    let mut tokens = sql.split_whitespace().map(|s| s.to_lowercase());
+    fn next_token(tokens: &mut impl Iterator<Item = String>) -> String {
+        tokens.next().unwrap()
+    }
+    match next_token(&mut tokens).as_str() {
+        "create" => match next_token(&mut tokens).as_str() {
+            "materialized" => {
+                // view
+                tokens.next();
+                // name
+                let name = next_token(&mut tokens).to_string();
+                SqlCmd::CreateMaterializedView { name }
+            }
+            _ => SqlCmd::Create {
+                is_create_table_as: is_create_table_as(sql),
+            },
         },
+        "set" => {
+            if sql.contains("background_ddl") {
+                let enable = sql.contains("true");
+                SqlCmd::SetBackgroundDdl { enable }
+            } else {
+                SqlCmd::Others
+            }
+        }
```

Comment:
As `SET` will be replayed after recovery, is it okay to directly fetch the value of `background_ddl` with SQL?